### PR TITLE
Discovery Prompt, Notifications, and minor UI tweaks

### PR DESCRIPTION
**Pull request by zkayyali812**
Signed-off-by: Zachary Kayyali <zkayyali@redhat.com>

open-cluster-management/backlog#6600
open-cluster-management/backlog#6590

### CODE REVIEW

File: frontend/public/locales/en/discovery.json [hunk 1]
```diff
@@ -7,6 +7,12 @@
     "discovery.configureDiscovery": "Configure discovery settings",
     "discovery.addDiscovery": "Create discovery settings",
 
+    "alert.created.header": "{{credentialName}} discovery setting has been created successfully",
```
Comment (swopebe) [inline frontend/public/locales/en/discovery.json:10]: Please change "has been" to "was".

Comment (swopebe) [reply frontend/public/locales/en/discovery.json:10]: this applies to all "has been" statements below.

**Review by jamestalton — APPROVED** [review]
lgtm

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**